silk-plugins/silk-plugins-xml/src/main/scala/org/silkframework/plugins/dataset/xml/XmlTraverser.scala

```diff
@@ -96,8 +97,7 @@ case class XmlTraverser(node: Node, parentOpt: Option[XmlTraverser] = None) {
     // Generate paths for all attributes
     val attributes = if(depth == 0) Seq() else node.attributes.asAttrMap.keys.toSeq
     val attributesPaths = attributes.map(attribute => TypedPath((path :+ ForwardOperator("@" + attribute)).toList, StringValueType, isAttribute = true))
-    val pathValueType: ValueType = if(children.nonEmpty || node.attributes.nonEmpty) UriValueType else StringValueType
-    val typedPath = TypedPath(path.toList, pathValueType, isAttribute = false)
+    val typedPath = TypedPath(path.toList, UntypedValueType, isAttribute = false) //TODO TypedPath change: removed buggy String/Uri type distinction PAY SPECIAL ATTENTION TO THIS SECTION!
```

Comment:
Not a bug. If this has caused any problems further down the line, these problems should be fixed instead.